src/lib.rs

```diff
@@ -510,6 +510,7 @@ pub fn format_input<T: Write>(
     let silent_emitter = Box::new(EmitterWriter::new(
         Box::new(Vec::new()),
         Some(codemap.clone()),
+	false,
```

Comment:
Could you please fix the indentation here?

Reply:
Oups, my bad.